components/Collections/src/private/MDCCollectionViewEditor.m

```diff
@@ -472,9 +472,8 @@ - (void)panToReorderWithRecognizer:(UIPanGestureRecognizer *)recognizer {
         _reorderingCellIndexPath = newIndexPath;
 
         // Notify delegate that item will move.
-        if ([_delegate respondsToSelector:@selector(collectionView:
-                                              willMoveItemAtIndexPath:
-                                                          toIndexPath:)]) {
+        if ([_delegate respondsToSelector:@selector
```

Comment:
I feel like this change (I've seen it before) makes the code a bit harder to read. Does it adhere to the formatting guidelines?